app/questionnaire/location.py

```diff
@@ -0,0 +1,71 @@
+from flask import url_for
+
+
+class Location(object):
+
+    def __init__(self, group_id, group_instance, block_id):
+
+        self.group_id = group_id
+        self.group_instance = group_instance
+        self.block_id = block_id
+
+    def __eq__(self, other):
+        """
+        Check to see if two locations are equal.
+        Two answers are considered to be equal if their dictionary representations equal one another.
+
+        :param other: An answer to compare
+        :return: True if both answers match, otherwise False.
+        """
+        return self.__dict__ == other.__dict__
```

Comment:
Sorry, arbitrary links on the net does nothing for me: http://stackoverflow.com/questions/1227121/compare-object-instances-for-equality-by-their-attributes-in-python